src/imageview.cpp

```diff
@@ -88,6 +93,34 @@ void ImageView::mouseDoubleClickEvent(QMouseEvent* event) {
   QAbstractScrollArea::mouseDoubleClickEvent(event);
 }
 
+void ImageView::mousePressEvent(QMouseEvent * event) {
+  QGraphicsView::mousePressEvent(event);
+  if(cursorTimer_) cursorTimer_->stop();
+}
+
+void ImageView::mouseReleaseEvent(QMouseEvent* event) {
+  QGraphicsView::mouseReleaseEvent(event);
+  if(cursorTimer_) cursorTimer_->start(3000);
```

Comment:
If there would be a constant for the milliseconds then only one place would need an update if the value should be changed later. :)